examples/use-kits.yaml

```diff
@@ -0,0 +1,89 @@
+# Example: Use of non-serialized resources in quantity (ouikits)
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - ouikit: http://www.ouikit.org/
+
+'@id': rgh:valueflows/valueflows/master/examples/use-kits.yaml
+'@graph':
+
+  # recipe
+
+  - '@id': ouikit:a7e94dde-2997-451e-a7e1-8d364d5e6dee
+    '@type': ResourceSpecification
+    name: Ouikits
+    skos:note: An eating kit containing plate, bowl, cup, utensils.
+    unit: unit:Number
+    substitutable: True
+
+  - '@id': ouikit:c404cff5-19c7-453d-b0d8-d8e96055bd0b
+    '@type': RecipeProcess
+    skos:note: Hold a workshop with meal
+    time:hasDuration:
+      time:unitType: time:unitHour
+      time:numericDuration: 4
+    scalingFormula: D
+
+  - '@id': ouikit:ff208521-9fec-4bb4-a6d3-197f6735cb26
+    '@type': RecipeFlow
+    action: use
+    resourceConformsTo: ouikit:a7e94dde-2997-451e-a7e1-8d364d5e6dee  # reusable eating kit
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+    alsoQuantifiedAs:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 1
```

Comment:
why not `time:hasDuration` ?

Reply:
Because I want to use it as a quantity, with other quantities, in formulas that understand quantities and queries that understand quantities.  It doesn't matter that it could also be expressed as a duration.

Reply:
`time:hasDuration` also references a quantitative value, one can use it with formulas that understand quantities and queries that understand quantities. Besides all that it has very clear meaning while *quantifiedAs*, *alsoQuantifiedAs*, *additionallyQuantifiedAs* have no meaning.